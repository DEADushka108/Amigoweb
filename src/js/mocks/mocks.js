export * from './const/const';
export * from './store/store';
export * from './user/user';
