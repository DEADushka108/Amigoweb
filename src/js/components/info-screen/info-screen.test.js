import React from 'react';
import {Provider} from 'react-redux';
import renderer from 'react-test-renderer';
import {store, userInfo} from '../../mocks/mocks';
import InfoScreen from './info-screen';

it(`Should render InfoScreen component correctly`, () => {
  const tree = renderer
    .create((
      <Provider store={store}>
        <InfoScreen
          userInfo={userInfo}
        />
      </Provider>
    ), {
      createNodeMock() {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
