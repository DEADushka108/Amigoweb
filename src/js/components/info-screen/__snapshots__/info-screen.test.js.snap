// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render InfoScreen component correctly 1`] = `
Array [
  <h1>
    Отправленные данные
  </h1>,
  <div
    className="select__options"
  >
    <p
      className="select__option"
    >
      Имя: 
      batman
    </p>
    <p
      className="select__option"
    >
      Почта: 
      batman@batman.net
    </p>
    <p
      className="select__option"
    >
      Телефон: 
      89000000000
    </p>
    <p
      className="select__option"
    >
      Язык: 
      English
    </p>
  </div>,
]
`;
